pubnub-kotlin/pubnub-kotlin-api/src/iosMain/kotlin/com/pubnub/api/endpoints/FetchMessages.ios.kt

```diff
@@ -77,6 +77,8 @@ open class FetchMessagesImpl(
     }
 
     private fun mapMessageActions(rawValue: List<*>): Map<String, Map<String, List<PNFetchMessageItem.Action>>>? {
+        println(rawValue)
```

Comment:
clean up printlns

Reply:
Sure, I'll remove it and run these two commands you mentioned